webapp/src/domain/Place.tsx

```diff
@@ -2,29 +2,26 @@
  * Class to define a point retrieved from the SOLID dataset
  */
 export default class Place {
-    private _uuid: string | undefined;
+    private _uuid: string = crypto.randomUUID();
     private _title: string;
     private _latitude: number;
     private _longitude: number;
     private _description: string;
     private _photos: File[];
   
-    constructor(name: string, latitude: number, longitude: number, description: string, photos: File[]) {
+    constructor(name:string, latitude:number, longitude:number, description:string, photos:File[]=new Array<File>, id:string|undefined=undefined) {
       this._title = name;
       this._latitude = latitude;
       this._longitude = longitude;
       this._description = description;
       this._photos = photos;
+      this._uuid = (id === undefined) ? crypto.randomUUID() : id;
     }
 
-    get uuid(): string | undefined {
+    get uuid(): string {
         return this._uuid;
     }
 
-    set uuid(name: string | undefined) {
```

Comment:
Although generating the UUID here it's OK I think the setter property shouldn't be deleted as, if I'm not wrong, it should be recovered when the place is got from the POD

Reply:
Yes, that's why there is an optional id parameter in the constructor. After the data is recovered from the POD, the uuid is passed as an argument when creating the object.

Reply:
Oh OK, I didn’t see it